src/components/Match/matchColumns.jsx

```diff
@@ -622,6 +622,12 @@ export const inflictorsColumns = [
   },
 ];
 
+const colorizeDeaths = obj => (row, col, field) => (
```

Comment:
Initially duplicated another display function, forgot to clean up. Just wanted to open the PR before I left for the weekend.

Also need to update the text placement for the count of deaths.